examples/dynamic-config-cp/envoy.yaml

```diff
@@ -37,7 +37,6 @@ static_resources:
                 port_value: 18000
 
 admin:
-  access_log_path: /dev/null
```

Comment:
also some of these in the examples are possibly literally included

my suggestion woudl be to

```
git grep literalinclude docs/root/
```

to find them

Reply:
do we only care about 
```    
     :lines: 1-29
    :emphasize-lines: 13-14
```

Reply:
i think yes - but best check the output when done

the most recent ci build of docs for this pr can be seen:

https://storage.googleapis.com/envoy-pr/15461/docs/index.html

Reply:
Good catch though, I did not even think about this. Validated all of them and since the admin interface is last one to be added, all the highlights seems to be good.

Reply:
despite trying to think of some way to prevent these getting out of sync programatically i havent been able to - so it kinda requires knowing about the configs being used this way